Scripts/IncidentFields/script-IncidentFields.yml

```diff
@@ -0,0 +1,107 @@
+commonfields:
+  id: 0fc0b044-47fe-4d52-8a90-6e46a65c0389
+  version: 192
+name: IncidentFields
+script: |
```

Comment:
scripts should be packagified and this file should not exist

Reply:
Yep, forgot to delete it.  It's gone now.